### DISCUSSION TITLE
{"message":"Missing/invalid DB name","status":500}

### DISCUSSION THREAD

**Discussion by chadgriff** [Budibase/budibase#5573] [Help] [answered]
After updating Budibase to v 1.0.110 users are getting a {"message":"Missing/invalid DB name","status":500} message after they log in when they attempt to launch an app. My experience, as a builder, is that I am getting that error from time to time when attempting to view the published app. I have also noticed that whichever app was last published, is the app that gets presented regardless of which app you are selecting "View Published App". For example, if I have app1 and app2 published but app 2 was the most recently published app, whenever I select "View Published App" I am directed to app2 regardless if I am using the link for app1 or app2. While I don't have a screenshot, I noticed the app logging a Trace message of "data source not fully configured' when the Status 500 error was thrown.

I have not changed my data sources for either of the 2 apps that are published. I see issue  #5214 that was opened 14 days ago but not sure how to fully interpret the answer and how it might apply to my scenario.

Currently, both apps are not accessible by users so any guidance would be appreciated. If I need to supply logs please give me a short primer on where to access them as the only means I have currently is to use docker to attach to the bbapps container to watch the logging in real time.

**Comment by chadgriff** [comment]
Here is a log output which shows the error...
![2022-04-25_14-59-08](https://user-images.githubusercontent.com/15015476/165156179-6ec27e4c-cb6b-4317-bf97-74b740f0c793.png)

**Reply by melohagan** [reply]
Hi @chadgriff Were there any error logs in the **bbworker** service?

**Comment by chadgriff** [comment, marked as answer]
Melohagan,

When attached to the bbworker service there was no logging when the error is generated.

-Chad

Sent from my iPhone

On Apr 26, 2022, at 8:29 AM, melohagan ***@***.***> wrote:

﻿

Hi @chadgriff<https://github.com/chadgriff> Were there any error logs in the bbworker service?

**Reply by melohagan** [reply]
Have you tried removing the docker containers and doing another **docker-compose up** with the latest compose file?

**Reply by chadgriff** [reply]
@melohagan I removed the docker containers as you suggested and pulled the latest docker-compose file from the Budibase Docker documentation website. I re-created the containers using docker-compose up -d and it worked to resolve the issue.  Thanks for your help!

**Reply by melohagan** [reply]
Awesome, you're welcome!

**Reply by Bytelegion** [reply]
@melohagan please guide me how can i do the same for helm chart installation . Publics apps also not working on helm chart
